Asterix/main_THD.py

```diff
@@ -69,6 +69,7 @@ def runthd2(parameter_file_path,
             NewLoopconfig={},
             NewSIMUconfig={},
             silence=False,
+            save_probe=False,
```

Comment:
save_probes with an s would be a better name ? I'm not sure what you save actually so mybe not.